cluster/swarm/cluster.go

```diff
@@ -482,20 +482,26 @@ func (c *Cluster) CreateNetwork(request *types.NetworkCreate) (response *types.N
 	if err != nil {
 		return nil, err
 	}
-	if nodes != nil {
-		resp, err := c.engines[nodes[0].ID].CreateNetwork(request)
-		if err == nil {
-			if network := c.engines[nodes[0].ID].Networks().Get(resp.ID); network != nil && network.Scope == "global" {
-				for id, engine := range c.engines {
-					if id != nodes[0].ID {
-						engine.AddNetwork(network)
-					}
-				}
+	// no nodes selected
+	if nodes == nil {
```

Comment:
`SelectNodesForContainer` returns `errNoNodeAvailable` for this case so this is redundant.